pkg/bucketeer/api/api.go

```diff
@@ -27,13 +27,14 @@ func (c *client) GetEvaluation(req *model.GetEvaluationRequest) (*model.GetEvalu
 		req,
 	)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
 	}
 	var ger model.GetEvaluationResponse
 	if err := json.Unmarshal(resp, &ger); err != nil {
-		return nil, err
+		return nil, 0, err
 	}
-	return &ger, nil
+	fmt.Println(len(resp))
+	return &ger, len(resp), nil
```

Comment:
To keep consistency with the other SDKs, please use the `ContentLength` in the response from [here](https://github.com/ca-dp/bucketeer-go-server-sdk/blob/master/pkg/bucketeer/api/api.go#L72)